.github/CONTRIBUTING.md

```diff
@@ -99,58 +99,69 @@ If you got an idea for a component that you think should be part of Kirby you sh
 ## Setup environment and tooling for contribution
 <!-- The following section is responsible for teaching people how to get setup with 
 the repo such that they can get coding -->
-This section will help you get going with Kirby such that you can begin to contribute by writing code! 
+This section will help you get going with Kirby such that you can begin contributing. 
 
-#### Clone and install the repository
+### Clone and install the repository
 <!-- how to clone and get going with the kirby repo-->
 **Note:** make sure you have the right version of Node installed before installing the repository (see: [Node](#node)).
 
 1. Clone the repository: 
-`git clone https://github.com/kirbydesign/designsystem`
+```
+git clone https://github.com/kirbydesign/designsystem
+```
 
 2. Install dependencies: 
-`npm install`
+```
+npm install
+```
 
 3. Start the cookbook server:
-`npm start`
+```
+npm start
+```
 
-You should be able to access the Cookbook by opening `http://localhost:4200` in a browser.
+You should now be able to open `http://localhost:4200` in a browser to access the cookbook.
 
-#### Node 
+### Node 
 <!-- Which version should people use, and perhaps a recommendation on which tool people can use to manage versions. -->
 [Node](https://nodejs.org/en/) is the runtime used for development. 
+
 We recommend using Node version `INSERT HIGHEST VERSION WE HAVE GOTTEN RUNNING WITHOUT ISSUES`. This is because we have experienced errors with other version due to **What was the problem exactly again?**.  
 
 If you have need other versions of Node installed then a version manager such as [NVM](https://github.com/nvm-sh/nvm) can be used. 
 
-#### Tslint 
+### Tslint 
```

Comment:
Capital `S`

TSLint instead of Tslint.

Reply:
Should be fixed by new commits